dotCMS/src/main/java/com/dotcms/publisher/business/PublisherQueueJob.java

```diff
@@ -463,4 +494,34 @@ private PublisherConfig setUpConfigForPublisher(PublisherConfig pconf)
         return pconf;
     }
 
+    private class GroupPushStats {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1258)